NOTES.ANDROID

```diff
@@ -0,0 +1,66 @@
+
+ NOTES FOR THE ANDROID PLATFORMS
+ ===============================
+
+ Requirement details
+ -------------------
+
+ Beside basic tools like perl and make you'll need to download Android
+ NDK. It's available for Linux, Mac OS X and Windows, but only Linux
+ version was actually tested. There is no reason to believe that Mac OS X
+ wouldn't work. And as for Windows, it's unclear which "shell" would be
+ suitable, MSYS2 might have best chances. NDK version should play lesser
+ role, the goal is to support a range of most recent versions.
+
+ Configuration
+ -------------
+
+ Android is naturally cross-compiled target and you can't use ./config.
+ You have to use ./Configure and name your target explicitly, there are
+ android-arm, android-arm64, android-mips, android-mip64, android-x86
+ and android-x86_64. Do not pass --cross-compile-prefix (as you might
+ be tempted), as it will be "calculated" automatically based on chosen
+ platform. Though you'd still need to know it, the prefix, for $PATH
```

Comment:
Though you still might need to know the prefix to set PATH correctly, in order to invoke...

Reply:
I can't agree with "might", because there is no uncertainty. Otherwise I followed the spirit of suggestion. I also replaced "a bit messy" remark with "Configure will tell you if you get it wrong" and backed it up with a check in 15-android.conf ;-)